Header header
uint32 numJobsAccepted
uint32[] acceptedJobID
time[] jobEndTime
